minifi/minifi-nar-bundles/minifi-framework-bundle/minifi-framework-nar/pom.xml

```diff
@@ -79,6 +89,22 @@ limitations under the License.
             <artifactId>nifi-security-utils-api</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-security-utils</artifactId>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-framework-nar-loading-utils</artifactId>
+            <version>1.16.0-SNAPSHOT</version>
```

Comment:
As a general question (maybe not in scope for this change)  I saw that dependency version for 1.16 is scattered everywhere would it make sense to extract it so it could be updated / changed in only one place? What do you think?

Reply:
This is pretty standard across the components, IIRC it had something to do with using `${project.version}` causing issues. The Release Management scripts handle changing this version everywhere, when we do releases and update the main branch to the next snapshot version.